### PR TITLE
Take out of quarantine test_id:3145 for nonroot lanes

### LINKED ISSUE #6473: re-enable release update functional test for non-root test lanes.
https://github.com/kubevirt/kubevirt/pull/6472 disabled the update test for non-root in order to unblock PRs. non root is still experimental, so this shouldn't be considered a release blocker.

We still need to re-enable this test `test_id:3145` quickly for non-root given how important update guarantees are.

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
Fixed by https://github.com/kubevirt/kubevirt/pull/6445

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #6473

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -409,9 +409,8 @@ if [ -z "$KUBEVIRT_QUARANTINE" ]; then
     else
         export KUBEVIRT_E2E_SKIP="QUARANTINE"
     fi
-    # quarantine test_id:3145 only for nonroot lanes
```
Comment (enp0s3) [inline automation/test.sh:412]: @xpivarc Hi. I think you can remove the whole condition block:
```
    if [[ $KUBEVIRT_NONROOT =~ true ]]; then
        export KUBEVIRT_E2E_SKIP="${KUBEVIRT_E2E_SKIP}"
    fi
```

Comment (xpivarc) [reply automation/test.sh:412]: @enp0s3 done.

**Comment by xpivarc** [comment]
PTAL @maya-r @enp0s3

**Review by enp0s3 — APPROVED** [review]
Thank you!
/approve

**Pull request merged**